classes/Http/Controller/Admin/TalksController.php

```diff
@@ -86,7 +84,9 @@ public function viewAction(Request $request)
             return $this->app->redirect($this->url('admin_talks'));
         }
 
-        return $this->render('admin/talks/view.twig', ['talk' => $handler->getProfile()]);
```

Comment:
While at it, I figured it would make sense to streamline how we pass data to the templates - this is an example.